### PR TITLE
Updated dependencies + webpack plugin replacement for sed native-shim backtick modifier.

### PR DESCRIPTION
**Pull request by micahscopes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,16 +31,17 @@
...
     "@webcomponents/template": "https://github.com/webcomponents/template/archive/60d743e.tar.gz",
-    "cloudydom": "^1.0.1",
     "array.from": "^1.0.3",
+    "cloudydom": "^1.0.2",
     "es6-promise": "^4.0.5",
     "object.assign": "^4.0.4"
   },
   "devDependencies": {
     "cz-conventional-changelog": "1",
+    "fs": "^0.0.1-security",
```
Comment (treshugart) [inline package.json]: Why is this needed over just using the standard node `fs` module?

**Comment by micahscopes** [comment]
Ooh, it's probably not! I actually am not that familiar with node, so
didn't realize that was built in

File: package.json (hunk 1 above)
Comment (micahscopes) [reply package.json]: I've removed this dependency.

**Comment by micahscopes** [comment]
Btw, I considered using [nf](https://www.npmjs.com/package/nf) to do a more straightforward replacement of the shell stuff, but decided to go this more drawn out way.
